src/todos/todos.database.service.ts

```diff
@@ -1,27 +1,32 @@
 import { Injectable } from '@nestjs/common'
 import { InjectModel } from '@nestjs/mongoose'
 import { Model } from 'mongoose'
-import { NotFoundError } from 'src/common/errors/errors'
 import { TODO_MODEL } from 'src/common/constants/database.constants'
-import type { Todo } from './schemas/todos.schema'
+import { NotFoundError } from 'src/common/errors/errors'
 import { CreateTodoDto } from './dto/create-todo.dto'
 import { UpdateTodoDto } from './dto/update-todo.dto'
+import type { Todo } from './schemas/todos.schema'
 
 @Injectable()
 export class TodosDatabaseService {
   constructor(@InjectModel(TODO_MODEL) private todoModel: Model<Todo>) {}
 
-  async create(createTodoDto: CreateTodoDto): Promise<Todo> {
+  async createTodo(createTodoDto: CreateTodoDto): Promise<Todo> {
     const createdTodo = await this.todoModel.create(createTodoDto)
     return createdTodo.toObject()
   }
 
-  async getAll(): Promise<Todo[]> {
+  async getAllTodos(): Promise<Todo[]> {
     const allTodos = await this.todoModel.find()
     return allTodos.map((todo) => todo.toObject())
   }
 
-  async getById(id: string): Promise<Todo> {
+  async getAllTodosByUserId(userId: string): Promise<Todo[]> {
```

Comment:
Let's add some filters (time range, for example) and sorts. It will be necessary unless we want to display unlimited list in UI.
We can do it in the next PR.

Reply:
Comment for the next step added ✅